jupyter_console/_version.py

```diff
@@ -1,2 +1,10 @@
-version_info = (5, 0, 0, 'dev')
-__version__ = '.'.join(map(str, version_info))
+""" For beta/alpha/rc releases, the version number for a beta is X.Y.ZbN 
+**without dots between the last 'micro' number and b**. N is the number of
+the beta released i.e. 1, 2, 3 ...
+
+See PEP 440 https://www.python.org/dev/peps/pep-0440/
+"""
+
+version_info = (5, 0, 0, '.dev')
```

Comment:
Yeah, i'm trying to figure out how to make s/bdist fails if version number
wrong. But not prevent console to run...
On Jun 8, 2016 14:44, "Thomas Kluyver" <notifications@github.com> wrote:

> In jupyter_console/_version.py
> <https://github.com/jupyter/jupyter_console/pull/84#discussion_r66346176>:
>
> > @@ -1,2 +1,10 @@
> > -version_info = (5, 0, 0, 'dev')
> > -__version__ = '.'.join(map(str, version_info))
> > +""" For beta/alpha/rc releases, the version number for a beta is X.Y.ZbN
> > +**without dots between the last 'micro' number and b**. N is the number of
> > +the beta released i.e. 1, 2, 3 ...
> > +
> > +See PEP 440 https://www.python.org/dev/peps/pep-0440/
> > +"""
> > +
> > +version_info = (5, 0, 0, '.dev')
>
> In general, I think it's a bad idea to do things depending on the version
> number, because you don't expect changing a version number to ever affect
> how code runs.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jupyter/jupyter_console/pull/84/files/ff638239f029ef01e04992c89d69c91b2bec5aba..fec2031a30de19975ce3ad9a38a802cef79d464e#r66346176>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAUez7aYpK3e6Zd7QsmFGRjerWzQZhDTks5qJzfQgaJpZM4IwVt8>
> .
>